spec/fuzzy-finder-spec.js

```diff
@@ -734,6 +747,9 @@ describe('FuzzyFinder', () => {
           await projectView.toggle()
 
           expect(PathLoader.startTask).toHaveBeenCalled()
+
+          await waitForReCrawlerToFinish(projectView)
```

Comment:
There's a very small behaviour difference with this PR: before when adding or removing a project from the current Atom window the fuzzy finder was temporarily shown as empty while it was recrawling the new list of projects. Now, it displays the old results while the re-crawling happens (which is consistent with what happens when a new file is added/removed).

Before:

![before](https://user-images.githubusercontent.com/408035/55147876-7fca4100-5147-11e9-8c71-3c2993d9a5bd.gif)

Now:

![now](https://user-images.githubusercontent.com/408035/55147887-8658b880-5147-11e9-8c4c-7be91e5cab14.gif)

Based on its description, this test was not doing the assertion correctly: instead of checking that the end list of results was empty was just checking that the temporary loading window was empty (which also could cause some flakiness if the crawling happened faster than the time it takes to open the fuzzy finder), so I fixed it.